### ISSUE TITLE
Notice: Undefined index: imageover_subcats 

### ISSUE THREAD

**Issue by Cheaternl** [spotweb/spotweb#79]
This message is displayed many times in the spots overview.
It happened when I upgraded to the current latest version and then retrieving spots.

Notice: Undefined index: imageover_subcats in /volume1/web/spotweb/lib/services/Settings/Services_Settings_Container.php on line 62

Probably the setting doesn't exist yet after upgrading. A proper way of handling unknown settings should be applied. Default 'off'.
After enabling the 'Enable imagepreview in spot overview' + saving the settings from tab 'Performance' then disabling this setting, does fix the numerous messages for me. They are gone.

**Comment by Sweepr** [comment]
Run upgrade-db.php

**Comment by Cheaternl** [comment]
I am sorry not providing that information directly. I am using a synology NAS. I upgraded through the package manager. I wasn't aware of the upgrade-db.php script. I executed it, and this was the output:

MYNAS> php upgrade-db.php
Updating schema..(pdo_mysql)
        .... <cut>
        Validating debuglog(microtime)
                Modifying column microtime (charset) on debuglog
        .... <cut>
Schema update done
Updating settings
Settings update done
Updating users
Users' update done
Performing basic analysis of database tables
Basic database optimalisation done

The error message was gone already, but I have more confident in this upgrade script than the procedure that I did, so thanks anyway!
